src/arch/riscv/interrupts.c

```diff
@@ -14,11 +14,15 @@
 #include <vm.h>
 #include <arch/csrs.h>
 #include <fences.h>
+#include <arch/aclint.h>
 
 void interrupts_arch_init()
 {
     if (cpu()->id  == CPU_MASTER) {  
         irqc_init();
+        if(DEFINED(ACLINT_PRESENT)){
```

Comment:
Just from an aesthetics perspective, could we somehow rename the generated macro and create a new macro `#define ACLINT_PRESENT() DEFINED(ACLINT_BASE_ADDR)` so that:

```suggestion
        if(ACLINT_PRESENT()){
```

Something like this would be more readable.